src/ngraph/runtime/cpu/cpu_emitter.cpp

```diff
@@ -2965,7 +2965,8 @@ namespace ngraph
                 // TODO(jmenon): Optimize for 1D
 
                 // TODO(jmenon): Remove element type restriction
-                if (arg_rank == 4 && max_pool->get_window_shape().size() == 2 &&
+                if (((arg_rank == 4 && max_pool->get_window_shape().size() == 2) ||
```

Comment:
This should use use_mkldnn_kernel(). looks like we missed this last time.

Reply:
Fixed.